source-container-build/app/source_build.py

```diff
@@ -389,8 +391,11 @@ def build_and_push(
     ]
     log.debug("push source image %r", push_cmd)
     run(push_cmd, check=True)
-    with open(digest_file, "r") as f:
-        build_result["image_digest"] = f.read().strip()
+    try:
+        with open(digest_file, "r") as f:
+            build_result["image_digest"] = f.read().strip()
+    finally:
+        os.unlink(digest_file)
```

Comment:
Could you make this change in a separate commit?

Reply:
This change is removed. I'll open another PR for this change.